app/bundles/LeadBundle/Entity/CustomFieldEntityTrait.php

```diff
@@ -119,6 +119,10 @@ public function addUpdatedField($alias, $value, $oldValue = null)
 
         if (property_exists($this, $property) && method_exists($this, $setter)) {
             // Fixed custom field so use the setter but don't get caught in a loop such as a custom field called "notes"
+            // Set empty value as null
+            if (empty($value)) {
```

Comment:
This will catch values of 0

Reply:
Fixed.
The test should be similar like app\bundles\LeadBundle\Tests\Entity\CustomFieldRepositoryTraitTest.php ?
Have you already some stuff how to write these tests?